drivers/i2c/bme280_driver.go

```diff
@@ -113,6 +114,7 @@ func (d *BME280Driver) initHumidity() (err error) {
 	d.hc.h5 = 0 + (int16(addrE6) << 4) | (int16(addrE5) >> 4)
 
 	d.connection.WriteByteData(bme280RegisterControlHumidity, 0x3F)
+	d.connection.WriteByteData(bmp280RegisterControl, 0x3F)
```

Comment:
I had taken line 117 from the initialisation code that calls into the BMP280 driver's initialisation code. Re-reviewing the BMP280 and BME280 data sheets, a value of `0x3F` does *not* seem correct for either of the lines in question.

Line 117 above would be better replaced with this:
```
// The 'ctrl_hum' register sets the humidity data acquisition options of
// the device. Changes to this register only become effective after a write
// operation to 'ctrl_meas'. Read the current value in, then write it back
cmr, err := d.connection.ReadByteData(bmp280RegisterControl)
if err != nil {
	d.connection.WriteByteData(bmp280RegisterControl, cmr)
}
```

This is the approach taken by other BME280 drivers, including the official Bosch BME280 driver (https://github.com/BoschSensortec/BME280_driver/blob/master/bme280_support.c, lines 172 - 180) as it does not override existing oversampling rates that have been set up for the `ctrl_meas` register.

For the byte written to `bme280RegisterControlHumidity`, the value is the oversampling rate, as defined in Table 20 of the Bosch BME280 data sheet where basically three bits control an oversampling rate of x0 (skipped) to x16.

I would suggest a default oversampling rate of 1 (as per Section 3.5 of the data sheet: "Recommended modes of operation"). 

If you are happy with this approach, I will update the code.